src/reducer.js

```diff
@@ -802,6 +804,20 @@ export const onPushApiFailed = (state) => ({
   },
 });
 
+export const onPushReset = (state) => ({
+  ...state,
+  pushNotification: {
+    ...state.pushNotification,
```

Comment:
Why do you need this? You are resetting all the keys, aren't you?

Reply:
Maybe we should do something like 

```
pushNotification: initialState.pushNotification,
```